static/src/javascripts/projects/common/views/svg.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+export const addClassesAndTitle = (
+    markup: string,
+    classes?: ?Array<string>,
+    title?: ?string
+): string => {
+    // Only mess with classes if we actually need to.
+    let markupWithClasses = markup;
+    if (classes) {
+        if (Array.isArray(classes)) {
```

Comment:
I wasn't sure what the story was if it were still currently being called from somewhere without the magic:

`// @flow`

I guess it's probably safe to assume that anywhere currently using it is already doing it right and that new places would all use flow.